mi/sql_query/sql.py

```diff
@@ -1,14 +1,34 @@
 from mi.sql_query.model import Sql
+from mi.sql_query.psycopg2 import connection as Connection
+from mi.sql_query.psycopg2 import cursor as Cursor
 from mi.sql_query.psycopg2 import sql as psycopg2_sql
 
+STATEMENT_SEPARATOR = ";"
+SELECT = "SELECT"
+
 
 def _is_select_query(query: str):
-    return query.startswith("SELECT")
+    return query.startswith(SELECT)
+
 
+def execute_sql_query(cursor: Cursor, sql: Sql) -> list:
+    connection: Connection = cursor.connection
 
-def execute_sql_query(cursor, sql: Sql) -> list:
-    query = psycopg2_sql.SQL(sql.statement).format(
-        **{k: psycopg2_sql.Identifier(v) for k, v in sql.identifiers.items()}
+    # Need to split statements up since when using "autocommit"
+    # it is not possible to invoke multiple statements in a single call.
+    # In practice this is needed for the database clear-down in the
+    # database_administration.tf
+    statements: list[str] = (
+        filter(
```

Comment:
So in this case we only filter the statements on the if statement being true? Im not used to seeing it above the if itself haha